bundles/framework/divmanazer/component/FileInput.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @class Oskari.userinterface.component.FileInput
+ *
```

Comment:
Document usage instructions and the required localization here. Also a brief description would be nice.

Reply:
The individual methods would also benefit from some comments.